src/emulator/auth/handlers.ts

```diff
@@ -331,3 +345,21 @@ export function registerHandlers(
     );
   });
 }
+
+function createFakeClaims(info: ProviderUserInfo): string {
+  const claims: IdpJwtPayload = {
+    sub: info.rawId,
+    iss: "",
+    aud: "",
+    exp: 0,
+    iat: 0,
+    name: info.displayName,
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    screen_name: info.screenName,
+    email: info.email,
+    // eslint-disable-next-line @typescript-eslint/camelcase
+    email_verified: true, // TODO: Shall we allow changing this?
```

Comment:
Probably will want to allow people to change this; folks get confused when facebook accounts with gmail don't count as account_verified even though facebook does verify emails

Reply:
I'll bring with up with UX